packs/equipment-effects/effect-shining-symbol-weakness.json

```diff
@@ -0,0 +1,42 @@
+{
+    "_id": "MFbOXncTwYaskdfG",
+    "img": "icons/commodities/treasure/broach-jewel-gold-blue.webp",
+    "name": "Effect: Shining Symbol Weakness",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Shining Symbol], @UUID[Compendium.pf2e.equipment-srd.Item.Shining Symbol (Greater)], @UUID[Compendium.pf2e.equipment-srd.Item.Shining Symbol (Major)]</p>\n<p>Your enemies gain weakness to spirit damage.</p>"
+        },
+        "duration": {
+            "expiry": null,
+            "sustained": false,
+            "unit": "unlimited",
+            "value": -1
+        },
+        "level": {
+            "value": 3
+        },
+        "publication": {
+            "license": "ORC",
+            "remaster": true,
+            "title": "Pathfinder GM Core"
+        },
+        "rules": [
+            {
+                "key": "Weakness",
+                "type": "spirit",
+                "value": "@item.origin.flags.pf2e.shiningSymbolWeakness"
+            }
+        ],
+        "start": {
+            "initiative": null,
+            "value": 0
+        },
+        "tokenIcon": {
+            "show": false
```

Comment:
Decided not to show the token icon for weakness, as they will already have an icon from the penalty effect.